cardano-cli/src/Cardano/CLI/EraBased/Run/Genesis/CreateTestnetData.hs

```diff
@@ -253,6 +255,26 @@ runGenesisCreateTestNetDataCmd
 
     when (0 < numPools) $ writeREADME poolsDir poolsREADME
 
+    -- CC members
+    ccColdKeys <- forM [1 .. numCommitteeKeys] $ \index -> do
```

Comment:
Are we not missing a transient vs on disk distinction here?

Reply:
@Jimbo4350> Yes we do miss the distinction. I thought the transient case was not worth the extra effort for the constitutional committee, because I think having a committee with millions of members is not a case we likely want to spend resource testing.

WDYT? Do you want me to add a on-disk/transient distinction?

Reply:
I would check with @mgmeier

Reply:
Given the optional nature of creating constitutional committee keys, and the expected number of keys being vastly lower than for instnace for DReps, there's no need for a transient mode from Performance & Tracing.